Dockerfile

```diff
@@ -114,6 +114,8 @@ WORKDIR /app/packages/cardano-services
 CMD ["node", "dist/cjs/cli.js", "start-blockfrost-worker"]
 
 FROM cardano-services as pg-boss-worker
+WORKDIR /config
+COPY compose/schedules.json .
```

Comment:
is this the default search location or how is the application finding this path?

Also maybe we can just change it to cli/env argument? Otherwise its annoyingly difficult to tweak it without rebuilding image

Reply:
The file must be added, somehow and somewhere...
The path can be provided to `pg-boss-worker` through `SCHEDULES` environment variable.
I guess the required changes to specify the filed on a different path from the default one should be added to this PR.

Reply:
```suggestion
WORKDIR /config
COPY compose/schedules.json .
ENV SCHEDULES=/config/schedules.json
```

Reply:
Which is the reason for this change?
We set environment variables through docker compose configuration.

Reply:
Errata corrige

> I guess the required changes to specify the file on a different path from the default one should be added to this PR.

I double checked this and there is no default: the `SCHEDULES` environment variable **must be specified** for deployed `pg-boss-worker`.

Reply:
I think we should set it globally since it does not impact other services. This way it's less error-prone.
Also this is image is pg-boss-worker specific, thus I don't think that there are any issues with setting it here

Reply:
ba0838545e should solve this